tests/foreman/ui_airgun/test_contentview.py

```diff
@@ -1350,3 +1362,503 @@ def test_positive_update_filter_affected_repos(session, module_org):
                 packages[0]['Name'] == repo2_package_name
                 and packages[0]['Version'] == '5.6.6'
         )
+
+
+@tier2
+def test_positive_search_composite(session):
+    """Search for content view by its composite property criteria
+
+    :id: 214a721b-3993-4251-9b7c-0f6d2446c1d1
+
+    :customerscenario: true
+
+    :expectedresults: Composite content view is successfully found
+
+    :BZ: 1259374
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    composite_name = gen_string('alpha')
+    with session:
+        session.contentview.create({
+            'name': composite_name, 'composite_view': True})
+        assert session.contentview.search('composite = true')[0]['Name'] == composite_name
```

Comment:
Kinda risky - you're reusing existing org where some other composite cv may be found. Maybe try `composite_name in {ccv['Name'] for ccv in session.contentview.search('composite = true')}` instead?
Or query containing both name & composite.